asset-properties-widget/asset-properties-widget-view/asset-properties.component.html

```diff
@@ -0,0 +1,56 @@
+<ng-container>
+  <div class="card-block">
+    <div class="text-center" *ngIf="isLoading">
+      <c8y-loading></c8y-loading>
+    </div>
+
+    <ng-container *ngIf="!isLoading">
+      <div
+        class="card m-b-8"
+        *ngFor="let prop of customProperties"
+        [ngClass]="{ 'card-highlight': prop.isEdit }"
+      >
+        <div class="card-block " [ngClass]="{ 'p-b-0': prop.isEdit }">
+          <div class="d-flex p-b-8" *ngIf="!prop.isEdit">
+            <p title="{{ prop.label | translate | humanize }}" class="text-medium text-truncate">
+              {{ prop?.label | translate | humanize }}
```

Comment:
I dont think both translate and humanize will not work... refer core code

Reply:
incorporate